client/blocks/express-checkout/stripe-link-preview/index.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable max-len */
+import icon from './icon.svg';
+import './style.scss';
+
+const useStripeLinkPreview = () => (
+	<div className="stripe-link-preview">
+		<img src={ icon } alt="" />
+	</div>
+);
+
+export default useStripeLinkPreview;
```

Comment:
This "preview" element is used on the edit page screen. 

Go to **Pages** in your Admin dashboard and edit the Block checkout page. You will see this preview:

<img width="794" alt="Screenshot 2024-10-24 at 2 59 09 pm" src="https://github.com/user-attachments/assets/c8bdae06-86cc-4058-b2cb-689de8d33c2f">

The Apple and Google Pay buttons previews looks a little sketchy so I'll file an issue to update those.

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/issues/3541